### PR TITLE
add (basically) working backend route

### PR DESCRIPTION
**Pull request by martanman**

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -14,3 +14,5 @@ requests==2.28.1
 types-paramiko==2.12.0.1
 types-requests==2.28.11.5
 uvicorn==0.20.0
+PyPDF2==3.0.1
+python-multipart==0.0.6
```
Comment (imagine-hussain) [inline backend/requirements.txt:18]: is this import needed / used?

**Review by imagine-hussain — APPROVED** [review]
small qn on the dependency;
otherwise, have fixed the mypy stuff, looks good to merge

**Comment by imagine-hussain** [comment]
Changed target branch to`331-staging` so that we dont need to merge 331 back in.

File: backend/requirements.txt (hunk 1 above)
Comment (martanman) [reply backend/requirements.txt:18]: do u mean are the requirements necessary? yes

Comment (imagine-hussain) [reply backend/requirements.txt:18]: okay in that case, looks good to merge

**Pull request merged**